rocksdb_admin/admin_handler.cpp

```diff
@@ -719,8 +731,19 @@ bool AdminHandler::restoreDBHelper(const std::string& db_name,
   }
   std::unique_ptr<rocksdb::BackupEngine> backup_engine_holder(backup_engine);
 
+  std::vector<rocksdb::BackupInfo> backup_infos;
```

Comment:
How many backups are typically stored? Asking since this is a new operation and wondering if it'll affect performance.